macOS test failues for test_solve_continuous_are

Failures on the newly completing daily builds:

```
======================================================================
FAIL: test_solvers.test_solve_continuous_are((array([[  0.00000000e+00,   1.00000000e+00,   0.00000000e+00,
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/scipy/linalg/tests/test_solvers.py", line 293, in _test_factory
    assert_array_almost_equal(res, np.zeros_like(res), decimal=dec)
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 14 decimals

(mismatch 6.25%)
 x: array([[ -6.66133815e-15,  -9.32587341e-15,  -1.33226763e-15,
         -4.88498131e-15],
       [ -9.32587341e-15,  -1.19904087e-14,  -3.55271368e-15,...
 y: array([[ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.]])

======================================================================
FAIL: test_solvers.test_solve_continuous_are((array([[  0.00000000e+00,   4.00000000e-01,   0.00000000e+00,
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/scipy/linalg/tests/test_solvers.py", line 293, in _test_factory
    assert_array_almost_equal(res, np.zeros_like(res), decimal=dec)
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/Users/travis/build/MacPython/scipy-wheels/venv/lib/python2.7/site-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 10 decimals

(mismatch 6.25%)
 x: array([[  2.01543449e-10,   7.12108150e-11,   5.43845524e-12,
          2.44526344e-11],
       [  7.12108150e-11,   1.14477317e-11,  -1.07208686e-12,...
 y: array([[ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.]])
```

https://api.travis-ci.org/jobs/205184824/log.txt?deansi=true

It looks like these could be fixed by relaxing the test thresholds to 13 decimals and 9 decimals respectively, but are these acceptable?

--- a/scipy/linalg/tests/test_solvers.py
+++ b/scipy/linalg/tests/test_solvers.py
@@ -277,8 +277,8 @@ def test_solve_continuous_are():
     #
     # If the test is failing use "None" for that entry.
     #
-    min_decimal = (14, 12, 14, 14, 11, 6, None, 5, 7, 14, 14,
-                   None, 10, 14, 13, 14, None, 12, None, None)
+    min_decimal = (14, 12, 13, 14, 11, 6, None, 5, 7, 14, 14,
+                   None, 9, 14, 13, 14, None, 12, None, None)
 
     def _test_factory(case, dec):
         """Checks if 0 = XA + A'X - XB(R)^{-1} B'X + Q is true"""

